README.md

```diff
@@ -0,0 +1,26 @@
+# metrics-py
```

Comment:
Probably it's clearer if they match, reduces friction slightly. Don't feel that strongly.